envFiles/.env.devcontainer

```
@@ -139,4 +139,11 @@ REDIS_TEST_MAPPED_PORT=6378
 
 ############## frontend URL ##############
 
-FRONTEND_URL="http://localhost:4321"
\ No newline at end of file
+FRONTEND_URL="http://localhost:4321"
+
+##### OTEL and Observability #####4141
+OTEL_ENABLED="true" # make it "false" if don't want span logging
+OTEL_SERVICE_NAME="talawa-api"
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318/v1/traces #TODO: after integrating Observability tool
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)